src/transformStateWithClones.js

```diff
@@ -65,6 +65,36 @@
  */
 function transformStateWithClones(state, transforms) {
   // write code here
+  const stateCopy = Object.assign({}, state);
+  const transformedArr = [];
```

Comment:
откуда такие названия?)
```suggestion
  const states = [];
```
```suggestion
  const history = [];
```